src/hydro/hydro_driver.cpp

```diff
@@ -74,6 +79,78 @@ TaskStatus CalculateGlobalMinDx(MeshData<Real> *md) {
   return TaskStatus::complete;
 }
 
+// Calculate the 1D cooling rate profile in histogram bins
+TaskStatus CalculateCoolingRateProfile(MeshData<Real> *md) {
+#if 0
```

Comment:
Even with a no-op task, the bug persists. I think we can conclusively say it's not cooling-related, but task reduction-related.